package.json

```diff
@@ -74,10 +73,9 @@
     "postcss-loader": "~8.1.1",
     "prettier": "~3.2.5",
     "react-refresh": "^0.14.2",
-    "sass": "1.77.6",
-    "sass-loader": "~14.2.1",
     "style-loader": "~4.0.0",
```

Comment:
Question: Still the `style-loader` required since we're using the `css-loader` and `postcss-loader` packages?